code/components/voip-mumble/src/MumbleClient.cpp

```diff
@@ -732,15 +731,14 @@ void MumbleClient::SetListenerMatrix(float position[3], float front[3], float up
 
 void MumbleClient::SendVoice(const char* buf, size_t size)
 {
-	if ((msec() - m_lastUdp) > kUDPTimeout)
+	// If we don't have UDP then we should send the packets over a TCP tunnel
+	if (!m_hasUdp)

```

Comment:
since it's false at the start, will this send tcp packet on the start until it resolve the first udp ping ?